tensorflow/contrib/tensorrt/test/base_test.py

```diff
@@ -20,17 +20,19 @@
 
 import numpy as np
 
+from tensorflow.contrib.tensorrt.python import trt_convert
 from tensorflow.contrib.tensorrt.test import tf_trt_integration_test_base as trt_test
 from tensorflow.python.framework import constant_op
 from tensorflow.python.framework import dtypes
 from tensorflow.python.framework import ops
 from tensorflow.python.ops import array_ops
+from tensorflow.python.ops import math_ops
 from tensorflow.python.ops import nn
 from tensorflow.python.ops import nn_ops
 from tensorflow.python.platform import test
 
 
-class SimpleSingleEngineGraphDefTest(trt_test.TfTrtIntegrationTestBase):
+class SimpleSingleEngineTest(trt_test.TfTrtIntegrationTestBase):
```

Comment:
Is this rename in the class name just a style thing? Is it relevant to this PR?

Reply:
It's more like a style change related to the PR, we added new test cases below so this help to keep the naming pattern simple. Let me know if you want me to revert it.

Reply:
Okay.